src/components/pages/CategoryPage/ProductItem.tsx

```diff
@@ -24,14 +24,14 @@ import Moment from 'react-moment';
 import { ProductItem } from '../../../types';
 import placeholderImg from '/src/img/placeholder-image.png';
 import { useDispatch, useSelector } from 'react-redux';
-import { profileStateSelector } from '../ProfilePage/selector';
+import { profileStateSelector } from '../../../redux/profile/selector';
```

Comment:
we can specify the path to redax like this:
import { profileStateSelector } from 'redux/profile/selector'